utils/maintenance/Release_Notes_Command.php

```diff
@@ -5,6 +5,32 @@
 
 final class Release_Notes_Command {
 
+	/**
+	 * Packages excluded from contributor list generation.
+	 * @var array
```

Comment:
Nitpick: there should a blank comment line above the first `@tag`. (not a blocker as the Docs rules are currently not applied yet anyway)